src/python/tox.ini

```diff
@@ -0,0 +1,22 @@
+# Tox (http://tox.testrun.org/)
+
+[tox]
+skipsdist = TRUE
+envlist = py35,docs
+
+[testenv]
+deps=-rrequirements.txt
+commands=pip install -e .
+         pytest
+         bash -c "pip uninstall FogLAMP <<< y"
+
+# TODO fix it
```

Comment:
use `# FIXME:` (for TODO, we should use `# TODO:`) mention issue# if any!
this will help to capture